ui/app/mainui/Popups.qml

```diff
@@ -1253,6 +1253,7 @@ QtObject {
                     swapFormData: swapInputParamsForm
                     swapOutputData: SwapOutputData{}
                 }
+                loginType: root.rootStore.loginType
```

Comment:
can we please add this to the SwapStore, if there is no correct store for it yet?

Reply:
It is coming from the very top (shared) RootStore; do we really need to add it to the SwapStore as well?

Reply:
Yeah then I can't see why we should add it here? It's a common property for the whole app

Reply:
because all properties that go to Swap and currently going via the store or the adaptor and it would be nice ti keep that consistent.